notebooks/ir_scattered_light_calwf3_corrections/Correcting_for_Scattered_Light_in_IR_Exposures_Using_calwf3_to_Mask_Bad_Reads.ipynb

```diff
@@ -0,0 +1,726 @@
+    "ax1.tick_params(axis='both', labelsize = 10)\n",
+    "cbar1 = plt.colorbar(im1, ax=ax1)\n",
+    "cbar1.ax.tick_params(labelsize=10)\n",
+    "\n",
+    "\n",
+    "ax2 = fig.add_subplot(rows,columns,2)\n",
+    "ax2.set_title(\"Original FLT image\", fontsize=20)\n",
+    "im2 = plt.imshow(image_old, vmin=0.8, vmax=1.3, origin='lower',  cmap='Greys_r')\n",
+    "ax2.tick_params(axis='both', labelsize=10)\n",
+    "cbar2 = plt.colorbar(im2, ax=ax2)\n",
+    "cbar2.ax.tick_params(labelsize=10)\n",
+    "\n",
+    "plt.rc('xtick', labelsize=10) \n",
+    "plt.rc('ytick', labelsize=10) \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This new image was produced by `calwf3` after masking the first 5 reads (not including the zero read) in the RAW file, reducing the effective exposure time from 1403 to 1000 seconds. While the total exposure is reduced from 1403 seconds to 1000 seconds (thus decreasing the overall S/N of the image), the background in the reprocessed image is now uniform over the entire field of view. We can see that the new FLT image is free of the Earth limb scattered light visible in the old FLT image. \n",
+    "\n",
+    "We update the FLT header keywords \"EXPTIME\" (in the primary header) and \"SAMPTIME\" (in the science header) to reflect the new total exposure time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "image_new = fits.open(reprocessed_flt, mode ='update')\n",
+    "\n",
+    "ima_orig = fits.open(original_ima)\n",
+    "        \n",
+    "hdr = ima_orig[0].header\n",
+    "NSAMP = hdr['NSAMP']\n",
+    "hdr1 = ima_orig[1].header\n",
+    "integ_time = np.zeros(shape=(NSAMP))\n",
+    "for i in range(1, NSAMP+1):\n",
+    "        integ_time[i-1] = ima_orig[('TIME', i)].header['PIXVALUE']\n",
+    "integ_time = integ_time[::-1]\n",
+    "\n",
+    "dt = np.diff(integ_time)\n",
+    "\n",
+    "final_time = integ_time[-1]\n",
+    "\n",
+    "if (len(reads) > 0):\n",
+    "    for read in reads:\n",
+    "        index = len(integ_time)-read-1 #because the reads are stored in reverse order\n",
+    "        final_time -= dt[index]\n",
+    "        \n",
+    "        \n",
+    "print(f'The final exposure time after reprocessing is {final_time}.')\n",
+    "image_new[0].header['EXPTIME'] = final_time\n",
+    "image_new.flush()\n",
+    "image_new.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, note that the reprocessed FLT product now includes a larger sky background in science pixels corresponding to IR \"blobs\", regions of reduced sensitivity due particulate matter on the channel selection mechanism (CSM) mirror([section 7.5 of the WFC3 Data Handbook](https://hst-docs.stsci.edu/wfc3dhb/chapter-7-wfc3-ir-sources-of-error/7-5-blobs)). \n",
```

Comment:
... (CSM) mirror (section 7.5 ...
(remove the space in between mirror and the parentheses)